robocop/checkers/spacing.py

```diff
@@ -123,3 +124,28 @@ def visit_File(self, node):  # noqa
                 self.report("empty-lines-between-sections", empty_lines, self.empty_lines_between_sections,
                             lineno=section.end_lineno, col=0)
         super().visit_File(node)
+
+
+class InconsistentUseOfTabsAndSpacesChecker(VisitorChecker):
+    """ Checker for inconsistent use of tabs and spaces. """
+
+    rules = {
+        "1006": (
+            "inconsistent-use-of-tabs-and-spaces",
```

Comment:
What about shortening it a little bit to "mixed-tabs-and-spaces"?

Reply:
+2 for the shorter one.